src/app/models/classesForStudents.ts

```diff
@@ -0,0 +1,9 @@
+export class Class {
```

Comment:
use names of the files according to name convention: class.model

Reply:
Changed.